src/RA_Leaderboard.h

```diff
@@ -51,22 +51,63 @@ class RA_Leaderboard
     virtual void Submit(unsigned int nScore);
 
 private:
-    const ra::LeaderboardID		m_nID;			//	DB ID for this LB
-    ConditionSet			m_startCond;	//	Start monitoring if this is true
-    ConditionSet			m_cancelCond;	//	Cancel monitoring if this is true
-    ConditionSet			m_submitCond;	//	Submit new score if this is true
+    const ra::LeaderboardID m_nID;          //  DB ID for this LB
+    ConditionSet            m_startCond;    //  Start monitoring if this is true
+    ConditionSet            m_cancelCond;   //  Cancel monitoring if this is true
+    ConditionSet            m_submitCond;   //  Submit new score if this is true
 
-    bool					m_bStarted;		//	False = check start condition. True = check cancel or submit conditions.
+    bool                    m_bStarted;     //  False = check start condition. True = check cancel or submit conditions.
     bool                    m_bSubmitted;   //  True if already submitted.
 
-    MemValue				m_value;		//	A collection of memory addresses and values to produce one value.
-    MemValue				m_progress;		//	A collection of memory addresses, used to show progress towards completion.
-    MemValue::Format        m_nFormat;		//	A format to output. Typically "%d" for score or "%02d:%02d.%02d" for time
+    MemValue                m_value;        //  A collection of memory addresses and values to produce one value.
+    MemValue                m_progress;     //  A collection of memory addresses, used to show progress towards completion.
+    MemValue::Format        m_nFormat;      //  A format to output. Typically "%d" for score or "%02d:%02d.%02d" for time
 
-    std::string				m_sTitle;		//	The title of the leaderboard
-    std::string				m_sDescription;	//	
+    std::string             m_sTitle;       //  The title of the leaderboard
+    std::string             m_sDescription; //  
 
-    std::vector<Entry>	    m_RankInfo;		//	Recent users ranks
+    std::vector<Entry>      m_RankInfo;     //  Recent users ranks
```

Comment:
Turned on a setting that lets me look at the spacing visually. The were all tabs.